### PR TITLE
zwesecur-remove-comments

### PR DESCRIPTION
**Pull request by John-A-Davies**
Signed-off-by: John Davies <daviesja@uk.ibm.com>

Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [ ] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [ ] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes 

#### Changes proposed in this PR

- Remove comment flags to enact the creation of all necessary objects
- Correct names
- Correct ZWES.IS

#### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *

---
* Description: _1 line description_ *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number:

### CODE REVIEW

File: files/jcl/ZWESECUR.jcl [hunk 1]
```diff
@@ -1,442 +1,443 @@
...
+//*                                                                             
+//*********************************************************************         
+//         EXPORT SYMLIST=*                                                     
+//*                                                                             
+//         SET PRODUCT=RACF          * RACF, ACF2, or TSS                       
+//*                     12345678                                                
+//         SET ADMINGRP=ZWEADMIN     * group for Zowe administrators            
+//         SET STCGROUP=&ADMINGRP.   * group for Zowe started tasks             
+//         SET ZOWEUSER=ZWESVUSR     * userid for Zowe started task             
+//         SET ZSSUSER=ZWESIUSR      * userid for xmem started task
```
Comment (stevenhorsman) [inline files/jcl/ZWESECUR.jcl]: This should not be changed to ZSSUSER - ZSS is a component that runs under ZWESVUSR. Please revert this

File: files/jcl/ZWESECUR.jcl [hunk 2]
```diff
@@ -1,442 +1,443 @@
...
+//*                                                                             
+//         SET PRODUCT=RACF          * RACF, ACF2, or TSS                       
+//*                     12345678                                                
+//         SET ADMINGRP=ZWEADMIN     * group for Zowe administrators            
+//         SET STCGROUP=&ADMINGRP.   * group for Zowe started tasks             
+//         SET ZOWEUSER=ZWESVUSR     * userid for Zowe started task             
+//         SET ZSSUSER=ZWESIUSR      * userid for xmem started task             
+//         SET  AUXUSER=&ZSSUSER.    * userid for xmem AUX started task         
+//         SET  ZOWESTC=ZWESVSTC     * Zowe started task name                   
+//         SET  ZSSSTC=ZWESISTC      * xmem started task name
```
Comment (stevenhorsman) [inline files/jcl/ZWESECUR.jcl]: Same as above. Please revert this

File: files/jcl/ZWESECUR.jcl (hunk 1 above)
Comment (Joe-Winchester) [reply files/jcl/ZWESECUR.jcl]: Agreed - the zss server is a C server that is part of the main ZWESVSTC started task.  It calls the cross memory server ZWESISTC, but there is no concept of a zss user.  There is the concept of a cross memory server user so XMEMUSER is better to use as the variable name in the JCL as this is what I've also called it in documentation

Comment (John-A-Davies) [reply files/jcl/ZWESECUR.jcl]: I've made those changes now.

File: files/jcl/ZWESECUR.jcl (hunk 2 above)
Comment (John-A-Davies) [reply files/jcl/ZWESECUR.jcl]: Done.

**Comment by stevenhorsman** [comment]
Now the defaults have been changed, should we update the comments now? eg `/** uncomment to use SUPERUSER.FILESYS, see JCL comments             */` -> `/** comment to NOT use SUPERUSER.FILESYS, see JCL comments             */`

**Comment by John-A-Davies** [comment]
I've adjusted the comments to reflect what's no longer commented out.

File: files/jcl/ZWESECUR.jcl [hunk 3]
```diff
@@ -1,488 +1,442 @@
...
-  LISTDSD PREFIX(&HLQ.) ALL
-  ADDSD  '&HLQ..*.**' UACC(READ) DATA('Zowe')
-  PERMIT '&HLQ..*.**' CLASS(DATASET) ACCESS(ALTER) ID(&SYSPROG.)
-
-  SETROPTS GENERIC(DATASET) REFRESH
-
-/* show results .................................................... */
-  LISTGRP &HLQ.         ALL
-  LISTDSD PREFIX(&HLQ.) ALL
-
```
Comment (stevenhorsman) [inline files/jcl/ZWESECUR.jcl]: Why have you deleted Onno's new changes!?

File: files/jcl/ZWESECUR.jcl [hunk 4]
```diff
@@ -1,488 +1,442 @@
...
-//* 10) Update the SET AUXSTC= statement to match the desired
-//*     ZSS Auxilary started task name.
-//*
-//* 11) Update the SET HLQ= statement to match the desired
-//*     Zowe data set high level qualifier.
-//*
-//* 12) Update the SET SYSPROG= statement to match the existing
-//*     user ID or group used by z/OS system programmers.
-//*
-//* 13) Customize the commands in the DD statement that matches your
```
Comment (stevenhorsman) [inline files/jcl/ZWESECUR.jcl]: Why have you deleted this changes!?

File: files/jcl/ZWESECUR.jcl [hunk 5]
```diff
@@ -1,488 +1,442 @@
...
+//ZWESECUR JOB                                                                  
+//* 
+//* This program and the accompanying materials are made available              
+//* under the terms of the Eclipse Public License v2.0 which                    
+//* accompanies this distribution, and is available at                          
+//* https://www.eclipse.org/legal/epl-v20.html                                  
+//*                                                                             
+//* SPDX-License-Identifier: EPL-2.0                                            
+//*                                                                             
+//* Copyright Contributors to the Zowe Project. 2018, 2019
```
Comment (stevenhorsman) [inline files/jcl/ZWESECUR.jcl]: Why have you changed the copyright dates back to 2019?

File: files/jcl/ZWESECUR.jcl [hunk 6]
```diff
@@ -1,488 +1,442 @@
...
-//         SET ADMINGRP=ZWEADMIN     * group for Zowe administrators
-//         SET STCGROUP=&ADMINGRP.   * group for Zowe started tasks
-//         SET ZOWEUSER=ZWESVUSR     * userid for Zowe started task
-//         SET XMEMUSER=ZWESIUSR     * userid for xmem started task
-//         SET  AUXUSER=&XMEMUSER.   * userid for xmem AUX started task
-//         SET  ZOWESTC=ZWESVSTC     * Zowe started task name
-//         SET  XMEMSTC=ZWESISTC     * xmem started task name
-//         SET   AUXSTC=ZWESASTC     * xmem AUX started task name
-//         SET      HLQ=ZWE          * data set high level qualifier
-//         SET  SYSPROG=&ADMINGRP.   * system programmer user ID/group
```
Comment (stevenhorsman) [inline files/jcl/ZWESECUR.jcl]: Why have you deleted this change?

File: files/jcl/ZWESECUR.jcl [hunk 7]
```diff
@@ -1,488 +1,442 @@
...
-/*            it on a production system.                             */
-
-/*TODO ACF2 permit zss UPDATE to FACILITY BPX.DAEMON & BPX.SERVER    */
-
-/* DEFINE ZOWE DATA SET PROTECTION ................................. */
-
-/* - &HLQ..SZWEAUTH is an APF authorized data set. It is strongly    */
-/*   advised to protect it against updates.                          */
-
-/*TODO ACF2 dataset protection, permit sysprog ALTER                 */
```
Comment (stevenhorsman) [inline files/jcl/ZWESECUR.jcl]: Why have you deleted this change?

File: files/jcl/ZWESECUR.jcl [hunk 8]
```diff
@@ -1,488 +1,442 @@
...
-  TSS PER(&ZSSUSER.) IBMFAC(BPX.DAEMON) ACC(UPDATE)
-  TSS WHOHAS IBMFAC(BPX.SERVER)
-  TSS PER(&ZSSUSER.) IBMFAC(BPX.SERVER) ACC(UPDATE)
-
-/* DEFINE ZOWE DATA SET PROTECTION ................................. */
-
-/* - &HLQ..SZWEAUTH is an APF authorized data set. It is strongly    */
-/*   advised to protect it against updates.                          */
-
-/*TODO TSS dataset protection, permit sysprog ALTER                  */
```
Comment (stevenhorsman) [inline files/jcl/ZWESECUR.jcl]: Why have you deleted this change?

**Comment by stevenhorsman** [comment]
Why have all the lines been padded with whitespace? This is just causing diff issues?

File: files/jcl/ZWESECUR.jcl (hunk 3 above)
Comment (John-A-Davies) [reply files/jcl/ZWESECUR.jcl]: I've re-instated Onno's new changes.

**Comment by stevenhorsman** [comment]
Superceded by #998

**Pull request closed without merging**